src/content/learn/removing-effect-dependencies.md

```diff
@@ -49,23 +49,23 @@ function ChatRoom({ roomId }) {
     const connection = createConnection(serverUrl, roomId);
     connection.connect();
     return () => connection.disconnect();
-  }, []); // <-- Fix the mistake here!
-  return <h1>Welcome to the {roomId} room!</h1>;
+  }, []); // <-- Corrija o erro aqui!
+  return <h1>Bem-vindo(a) à sala {roomId}!</h1>;
 }
 
 export default function App() {
-  const [roomId, setRoomId] = useState('general');
+  const [roomId, setRoomId] = useState('geral');
```

Comment:
We avoid translating the code values, it seems like I'm translating a variable hehe
```suggestion
  const [roomId, setRoomId] = useState('general');
```